src/darwin/Framework/CHIP/ChipBleDelegate.h

```diff
@@ -0,0 +1,131 @@
+/**
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2015-2017 Nest Labs, Inc.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file defines ChipBleDelegate interface
+ *
+ */
+
+#import "CHIPError.h"
+#import <CoreBluetooth/CoreBluetooth.h>
+
+@class CHIPDeviceController;
+
+typedef void (^PreparationCompleteHandler)(CHIPDeviceController * dm, CHIP_ERROR err);
+
+@interface ChipBleDelegate : NSObject <CBPeripheralDelegate>
+
+/*
+ * \defgroup BlePlatformDelegate
+ * @{
+ */
+
+/*
+ *  @brief Command from BleLayer to subscribe to a characteristic
+ */
+- (bool)SubscribeCharacteristic:(id)connObj service:(const CBUUID *)svcId characteristic:(const CBUUID *)charId;
+
+/*
+ *  @brief Command from BleLayer to unsubscribe a characteristic
+ */
+- (bool)UnsubscribeCharacteristic:(id)connObj service:(const CBUUID *)svcId characteristic:(const CBUUID *)charId;
+
+/*
+ *  @brief Command from BleLayer to close the underlying BLE connection
+ */
+- (bool)CloseConnection:(id)connObj;
+
+/*
+ *  @brief Command from BleLayer to retrieve the current MTU of the BLE connection
+ */
+- (uint16_t)GetMTU:(id)connObj;
+
+/*
+ *  @brief Command from BleLayer to send indication
+ */
+- (bool)SendIndication:(id)connObj service:(const CBUUID *)svcId characteristic:(const CBUUID *)charId data:(const NSData *)buf;
```

Comment:
Please lowercase all first characters of method signatures

Reply:
I can not. Those are names expected by the platform.